lhotse/features/kaldi/layers.py

```diff
@@ -160,6 +160,9 @@ def forward(
                 x, self._length, self._shift, self.snip_edges
             )
             log_energy = _get_log_energy(x_strided, self.energy_floor)  # size (m)
+        else:
+            # placeholder for torchscript
+            log_energy = torch.zeros_like(x)
```

Comment:
You can use `None` if you explicitly annotate the types;

function output -> `Tuple[torch.Tensor, Optional[torch.Tensor]]`
variable -> `log_energy: Optional[torch.Tensor] = None`

Reply:
adding